code/obj/item/organs/skull.dm

```diff
@@ -132,7 +132,18 @@
 
 		if (istool(W, TOOL_SAWING))
 			user.visible_message(SPAN_NOTICE("[user] hollows out [src]."))
-			var/obj/item/clothing/mask/skull/smask = new /obj/item/clothing/mask/skull
+			var/obj/item/clothing/mask/skull/M
```

Comment:
Please don't use non-standard single letter variable names, `M` is only acceptable as a variable name if it's a mob typed var.